partner_type_base/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013 Nicolas Bessi (Camptocamp SA)
+# Copyright 2014 Agile Business Group (<http://www.agilebg.com>)
+# Copyright 2015 Grupo ESOC (<http://www.grupoesoc.es>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner Address Types",
+    "summary": "Base implementation to improve the address type customization.",
+    "version": "18.0.1.0.0",
+    "author": "CRogos, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "maintainer": "glueckkanja AG",
```

Comment:
I think that you should put your github login here and the name of your company in author